.github/ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,8 @@ This is a (multiple allowed):
 * [ ] feature-discussion (RFC)
 
 ## DESCRIPTIVE TITLE HERE
```

Comment:
My personal thought is that it'll be ignored, but is better than nothing. As long as it doesn't get annoying (all caps top of every issue). Worth a try!